cloudbuild_docker.sh

```diff
@@ -26,47 +26,25 @@ docker_manifest() {
 }
 
 for distro_suffix in "" -debian9 -debian10; do
-  docker_manifest gcr.io/$PROJECT_ID/static${distro_suffix}:nonroot "amd64 arm64 s390x ppc64le"
-  docker_manifest gcr.io/$PROJECT_ID/static${distro_suffix}:latest "amd64 arm64 s390x ppc64le"
+  docker_manifest gcr.io/$PROJECT_ID/static${distro_suffix}:nonroot "amd64 arm arm64 s390x ppc64le"
+  docker_manifest gcr.io/$PROJECT_ID/static${distro_suffix}:latest "amd64 arm arm64 s390x ppc64le"
 
-  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:nonroot "amd64 arm64 s390x ppc64le"
-  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:latest "amd64 arm64 s390x ppc64le"
-  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:debug-nonroot "amd64 arm64 s390x ppc64le"
-  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:debug "amd64 arm64 s390x ppc64le"
+  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:nonroot "amd64 arm arm64 s390x ppc64le"
+  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:latest "amd64 arm arm64 s390x ppc64le"
+  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:debug-nonroot "amd64 arm arm64 s390x ppc64le"
+  docker_manifest gcr.io/$PROJECT_ID/base${distro_suffix}:debug "amd64 arm arm64 s390x ppc64le"
 
-  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:latest "amd64 arm64 s390x ppc64le"
-  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:debug "amd64 arm64 s390x ppc64le"
-done
-
-docker manifest create gcr.io/$PROJECT_ID/python3:nonroot \
-   gcr.io/$PROJECT_ID/python3:nonroot-amd64 \
-   gcr.io/$PROJECT_ID/python3:nonroot-arm64
-docker manifest push gcr.io/$PROJECT_ID/python3:nonroot
-
-docker manifest create gcr.io/$PROJECT_ID/python3:latest \
-   gcr.io/$PROJECT_ID/python3:latest-amd64 \
-   gcr.io/$PROJECT_ID/python3:latest-arm64
-docker manifest push gcr.io/$PROJECT_ID/python3:latest
+  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:latest "amd64 arm arm64 s390x ppc64le"
+  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:debug "amd64 arm arm64 s390x ppc64le"
```

Comment:
Just realized the PR also publishes manifest lists for `nonroot` and `debug-nonroot` for `cc`.

```suggestion
  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:nonroot "amd64 arm arm64 s390x ppc64le"
  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:latest "amd64 arm arm64 s390x ppc64le"
  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:debug-nonroot "amd64 arm arm64 s390x ppc64le"
  docker_manifest gcr.io/$PROJECT_ID/cc${distro_suffix}:debug "amd64 arm arm64 s390x ppc64le"
```

Reply:
added the nonroot variants for cc